.github/workflows/container-image.yml

```diff
@@ -1,51 +1,27 @@
-name: Build porta container image
+name: Container
 on:
-  # push:
-  #   branches:
-  #   - master
-  workflow_dispatch: {}
-  # pull_request:
-  # branches: [ master ]
-
-env:
-  # Use docker.io for Docker Hub if empty
-  REGISTRY: ghcr.io
-  # github.repository as <account>/<repo>
-  IMAGE_NAME: ${{ github.repository }}
+  push:
+    branches:
+      - master
+      - managed-services
+      - 3scale-[0-9]+-stable
+    tags:
+      - 3scale-[0-9]+-GA
```

Comment:
Same, regular expression doesn't match the point.

Reply:
see above